packages/web/src/services/Location.ts

```diff
@@ -26,6 +29,20 @@ export const getLocation = async (): Promise<Location | null> => {
     const res = await fetch('https://ipapi.co/json/')
     return res.json()
   } catch (e) {
+    console.debug(
```

Comment:
if you just do console.error it will go to sentry. will simplify this code a bit. we send console.error there

Reply:
nice